core/src/main/java/nl/nn/adapterframework/pipes/XmlValidator.java

```diff
@@ -492,6 +493,7 @@ public String getName() {
 		}
 
 		@IbisDoc({"name of the pipe", ""})
+		@Optional
```

Comment:
This change should not be necessary, as the ` @Optional` annotation should be inherited from `IValidator`